### ISSUE TITLE
Error on collectstatic in version 4.4.0

### ISSUE THREAD

**Issue by tom3rr** [jazzband/django-silk#574]
Since the update I get the following error when trying to run `python manage.py collectstatic`

```
whitenoise.storage.MissingFileError: The file 'silk/lib/"images/ui-icons_555555_256x240.png"' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x000001C0AA010C70>.

The CSS file 'silk\lib\jquery-ui-1.13.1.min.css' references a file which could not be found:
  silk/lib/"images/ui-icons_555555_256x240.png"

Please check the URL references in this CSS file, particularly any
relative paths which might be pointing to the wrong location.
```

It seems the css file is pointing to a missing asset.

**Comment by t-a-y-l-o-r** [comment]
downgrading to ==4.3 worked for me

**Comment by tom3rr** [comment]
The issue persist in version 5.0.0

**Comment by joshuadavidthomas** [comment]
This should be fixed in v4.4.1 and v5.0.1.
